helm/values.yaml

```diff
@@ -0,0 +1,17 @@
+domain:
+  url: ""
+stage:
+  image: quay.io/debezium/platform-stage:latest
+conductor:
+  image: quay.io/debezium/platform-conductor:nightly
+  offset:
+    existingConfigMap: ""
+kafka:
```

Comment:
Wondering if we should completely remove this. @jcechace, @jpechane WDYT?

Reply:
Yea.. What is this supposed to do? I would  deifinitively just create one -- there is no point in having this for helm (unlike the operator for specific DS instances).

Reply:
@mfvitale ok.. that I would remove for sure. 

However, my comment was actually for line 8 and the offset section -- I don't think we need the ability to use an  existing config map in this case.

Reply:
> However, my comment was actually for line 8 and the offset section -- I don't think we need the ability to use an existing config map in this case.

1. For consistency as we do for the secrets and also for DS operator. 
2. Maybe user want to control the life-cycle and not let the helm remove the offset during an `uninstall`. 

So I would keep it.

Reply:
1) This has nothing to do with DS operator... completely unrelate.
2) I'm not sure how much sense this would have -- considering it still depends on the WAL retention in the database and all..

However I don't have a strong opinion about it -- we can remove it when it shows to be redundant :)

Reply:
> I'm not sure how much sense this would have -- considering it still depends on the WAL retention in the database and all..

Currently this is not going to work since the `watcher` is configured without an explicit `snapshot.mode` and the default is `initial`. So if you remove the offset and the re-install the platform the outbox events will be reprocessed. Dunno what really happens on the DS instances, maybe the operator should be able to reconcile without recreating.

Reply:
> This has nothing to do with DS operator... completely unrelate.

Talking about a more general approach use in the let's call debezium ecosystem. 

> we can remove it when it shows to be redundant :)

And yes, we can always re-work this in future.